crates/ark/src/modules/positron/environment.R

```diff
@@ -14,15 +14,3 @@
 
     readLines(tf)
 }
-
-#' @export
-.ps.environment.describeCall <- function(expr, width.cutoff = 500L, nlines = -1L) {
```

Comment:
Moved from ark to harp, and renamed to `expr_deparse()`

(Needed to call it from harp tests directly)